wiki/Glossary/Downbeat/en.md

```diff
@@ -0,0 +1,12 @@
+---
+tags:
+  - onbeat
+---
+
+# Downbeat
+
+The downbeat is the impulse that occurs at the beginning of a bar in measured music. Its name is derived from the downward stroke of the director or conductor's baton on the first beat of each measure. It frequently carries the strongest accent of the rhythmic cycle.
```

Comment:
According to Wikipedia,
> The downbeat is the first beat of the bar, i.e. number 1. The upbeat is the last beat in the previous bar which immediately precedes, and hence anticipates, the downbeat.[5] Both terms correspond to the direction taken by the hand of a conductor.

This line should probably be rewritten; "impulse" is not the best term to describe this, but YMMV.

Also, I'm not so sure that "Downbeat" is something that needs to be described in a stub; it's a term used in music theory and would work better within something like the Music Theory articles. (But if you ask me, I don't really think those need to exist in the wiki.)

Reply:
agree, i'll rewrite this

imo. some duplicate content is fine so the user doesn't have to go down a huge chain to find the content they're looking for